lib/SILOptimizer/SILCombiner/SILCombinerMiscVisitors.cpp

```diff
@@ -781,24 +781,48 @@ SILCombiner::visitInjectEnumAddrInst(InjectEnumAddrInst *IEAI) {
 
   // Ok, we have a payload enum, make sure that we have a store previous to
   // us...
-  SILBasicBlock::iterator II = IEAI->getIterator();
+  auto allocStackOperand = IEAI->getOperand();
```

Comment:
Please follow the naming convention used in the rest of the file. In this case you should be using something like either:

AllocStackOperand or ASO.